src/Test/AbstractFixerTestCase.php

```diff
@@ -79,6 +89,10 @@ protected function getFixer()
 
         $this->fixer = $fixers[0];
 
+        if ($this->fixer instanceof SharedFixerConfigAwareInterface) {
+            $this->fixer->applySharedConfig($this->createSharedFixerConfig());
```

Comment:
default value should be done in fixed that is `SharedFixerConfigAwareInterface`, so if one is using package as lib he wont need to call the setter to avoid crash.